.travis.yml

```diff
@@ -1,4 +1,6 @@
 language: go
+go:
+  - 1.14.x
```

Comment:
Is there a reason 1.14.x is used here instead of 1.15.x?

Reply:
I don't think there is a particular reason. My suggestion would be to update it to 1.16 unless we want to also test backwards compatibility to old go versions. Then we could add them too. Either way, 1.16 should be in there.

Reply:
No reason. Actually I'm going to merge #23, to it doesn't matter anyways.

Reply:
Now I remember - the reason was, `go.mod` is using 1.14.

Reply:
Ah that makes sense. Unrelated to the current PR, what are the ramifications of updating go.mod to 1.15 or 1.16? cc @liamsi too

Reply:
As far as I remember this would indicate that we are using language features that were introduced with 1.16 (or whatever will be used in go.mod). Btw, these versions do not necessarily need to match (the one in go.mod and the one used for travis). The one in go.mod could be lower. In the IPFS Plugin case, I wanted all versions to match because the plug-in used some x/system package which caused problems. I don't think that's a problem here.